runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -13,27 +13,30 @@ namespace misc {
   // Helpers to convert certain unsigned symbols (e.g. Token::EOF) to their original numeric value (e.g. -1)
   // and vice versa. This is needed mostly for intervals to keep their original order and for toString()
   // methods to print the original numeric value (e.g. for tests).
-  size_t numericToSymbol(ssize_t v);
-  ssize_t symbolToNumeric(size_t v);
+  constexpr size_t numericToSymbol(ssize_t v) { return static_cast<size_t>(v); }
+
+  constexpr ssize_t symbolToNumeric(size_t v) { return static_cast<ssize_t>(v); }
```

Comment:
Semantically these two belong together and the leading comment applies to both. Hence there's no empty line in the original code.

Reply:
Done.